server/app/Console/Commands/DatabaseRebuild.php

```diff
@@ -3,28 +3,32 @@
 namespace App\Console\Commands;
 
 use Illuminate\Console\Command;
+use Symfony\Component\Console\Input\InputOption;
 
 class DatabaseRebuild extends Command
 {
     /**
-     * The console command name.
+     * The name and signature of the console command.
      *
      * @var string
      */
-    protected $signature = 'db:rebuild {--database=mysql}';
+    protected $signature = 'db:rebuild {--database=}';
```

Comment:
Looks like it does. This change breaks the `make db-reset` command. Fixing that now.